bpo-42819: disable Readline bracketed paste (#24108)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2021-01-04-23-54-34.bpo-42819.4KO6wU.rst

> Prevents bracketed paste from being enabled in the interactive interpreter,

Reviewer: ```suggestion
Prevent bracketed paste from being enabled in the interactive interpreter,
```

even if it's set in the inputrc or is enabled by default (eg GNU Readline

> 8.1). Patch by Dustin Rodrigues.

Reviewer 2: Hum, I suggest to say that Python now explicitly disables the bracketed mode when the readline module is imported, but also in the readline.read_init_file() function.

> Explicitly disable bracketed paste in the interactive interpreter, even if it's

Reviewer 2: ```suggestion
:mod:`readline`: Explicitly disable bracketed paste in the interactive interpreter, even if it's
```

set in the inputrc, is enabled by default (eg GNU Readline 8.1), or a user calls

> ``readline.read_init_file()``. The Python REPL has not implemented bracketed
> paste support so this prevents Readline defaults or global user configuration
> from causing unexpected behavior within Python. It can still be explicitly

Reviewer 2: ```suggestion
``readline.read_init_file()``. The Python REPL has not implemented bracketed
paste support. Also the bracketed mode writes `"\x1b[?2004h"` escape sequence into stdout
which causes test failures in applications which don't support it. It can still be explicitly
```

:mod:`readline`: Explicitly disable bracketed paste in the interactive 
interpreter, even if it's set in the inputrc, is enabled by default (eg GNU
Readline 8.1), or a user calls ``readline.read_init_file()``. The Python REPL
has not implemented bracketed paste support. Also, bracketed mode writes the

> `"\x1b[?2004h"` escape sequence into stdout which causes test failures in

Reviewer 2: Oh wait, I think that two ` are needed:

```suggestion
``"\x1b[?2004h"`` escape sequence into stdout which causes test failures in
```

applications that don't support it. It can still be explicitly enabled by

> calling ``readline.parse_and_bind("set enable-bracketed-paste on")``. This
> should be removed if bracketed paste mode is implemented (bpo-39820). Patch by

Reviewer 2: I don't think that the last sentence is useful for users reading the changelog.

```suggestion
calling ``readline.parse_and_bind("set enable-bracketed-paste on")``. Patch by
```
